Champions/Nidalee/W-C.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.Numerics;
+using System.Linq;
+using GameServerCore;
+using GameServerCore.Enums;
+using LeagueSandbox.GameServer.API;
+using GameServerCore.Domain.GameObjects;
+using static LeagueSandbox.GameServer.API.ApiFunctionManager;
+using GameServerCore.Domain;
+using LeagueSandbox.GameServer.Scripting.CSharp;
+using LeagueSandbox.GameServer.GameObjects.AttackableUnits.AI;
+
+namespace Spells
+{
+    public class Pounce : IGameScript
+    {
+        IChampion _championRef;
+        public void OnActivate(IChampion owner)
+        {
+        }
+
+        public void OnDeactivate(IChampion owner)
+        {
+        }
+
+        public void OnStartCasting(IChampion owner, ISpell spell, IAttackableUnit target)
+        {
+            
+        }
+
+        public void OnFinishCasting(IChampion owner, ISpell spell, IAttackableUnit target)
+        {                                    
+            var damage = owner.Spells[3].Level * 50f + owner.Stats.AbilityPower.Total * 0.3f;
+            var trueCoords = this.getTrueCoords(owner, spell, target);
+            //To have damage in location, so add a minion to do AOE
+            IMinion m = AddMinion(owner, "TestCube", "TestCube", trueCoords.X, trueCoords.Y);

```

Comment:
Do you sure `TestCube` is the right one to use?

Reply:
not sure, I use "TestCube" as the end of position.
and use it as AOE damage dealer